algorithms/matrix/multiply.py

```diff
@@ -0,0 +1,28 @@
+"""
+You are given 2 matrices A and B
+first it checks if the 2 martrices can be multiplied
+if it can be multiplied it multiplies them in O(n^3) time
+"""
+
+def multiply(A: list,B: list)->list:
+    rowsA,rowsB=len(A),len(B)
+    colsA,colsB=len(A[0]),len(B[0])
+    if(colsA != rowsB):
+        raise Exception("Matrices cannot be multipled")
+
+    result=[[0 for i in range(colsB)] for j in range(rowsA)]#create a result matrix
+    for i in range(rowsA):
+        for j in range(colsB):
+            for k in range(len(B)):
+                result[i][j] += A[i][k] * B[k][j]
+    return result
+
+if __name__ == "__main__":
```

Comment:
Can you write a unit test? See directory [tests](https://github.com/keon/algorithms/tree/master/tests). It exists for every directory in ```algorithms``` a py-file with tests.

Reply:
Yeah I guess that was the major issue, will be working on it, thanks